app/controllers/user_applications_controller.rb

```diff
@@ -126,16 +140,11 @@ def set_user_application
 
   def set_user_applications
     @user_applications =
-      UserApplication.joins("inner join users on users.id = user_applications.user_id")
-        .joins("inner join user_informations on users.id = user_informations.user_id")
-        .joins("inner join user_application_statuses on user_applications.user_application_status_id = user_application_statuses.id")
-        .select("user_applications.id, user_informations.first_name,
-          user_informations.last_name, user_applications.updated_at,
-          user_application_statuses.status,
-          (select title as c1 from Volunteer_Positions as v where v.id = first_choice_volunteer_position_id),
-          (select title as c2 from Volunteer_Positions as v where v.id = second_choice_volunteer_position_id),
-          (select title as c3 from Volunteer_Positions as v where v.id = third_choice_volunteer_position_id)")
-        .paginate(:page => params[:page], per_page: 100)
+      UserApplication.joins(set_user_applications_joins_users)
```

Comment:
@SeanLF Because this ultimately is getting a set of user_applications back, you might consider putting all this logic into the model.